src/app/components/engangsstonad/steps/Engangsstonad.step2.js

```diff
@@ -1,17 +1,70 @@
-// @flow
-import React from 'react';
+import React, { Component } from 'react';
+
+import { Normaltekst, Element } from 'nav-frontend-typografi';
+import { ToggleGruppe, ToggleKnapp } from 'nav-frontend-skjema';
 
-import { Normaltekst } from 'nav-frontend-typografi';
 import DialogBox from '../../shared/dialog-box/DialogBox';
+import DateInput from '../../shared/date-input/DateInput';
+import AttchmentList from '../../shared/attachment-list/AttachmentList';
+import AttchmentButton from '../../shared/attachment-button/AttachmentButton';
+
+class Step2 extends Component {
+    constructor(props) {
+        super(props);
+        this.toggleBirth = this.toggleBirth.bind(this);
+
+        this.state = {
+            childBorn: undefined
+        };
+    }
+
+    toggleBirth(e) {
+        if (e.target.value === 'childNotBorn') {
```

Comment:
Dette kan vel bare forenkles til this.setState({ childBorn: e.target.value === 'childNotBorn' });?

Reply:
Nesten! this.setState({ childBorn: e.target.value === 'childBorn' });

Men fikser.